src/main/resources/static/js/index.js

```diff
@@ -0,0 +1,134 @@
+$(function() {
```

Comment:
A third party library naming their charting "index" seems like they're asking for a filename conflict. All the more reason to put this in a sub-directory.